gulp/tasks/docs.js

```diff
@@ -39,7 +39,9 @@ gulp.task('generate-docs-json', cb => {
     paths.srcViews + '/**/*.js',
     '!' + paths.src + '/**/Style.js'
   ])
-    .pipe(g.plumber(err => {
+    // do not remove the function keyword
+    // we need 'this' scope here
+    .pipe(g.plumber(function(err) {
       g.util.log(err);
       this.emit('end');
```

Comment:
This was why watchers we dying on jsx parse error again.  `this.emit` was not the right `this`.  Arrow functions inherit `this` from the surrounding context, but we need a new scope here.